core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -160,7 +158,7 @@
           order: order,
           amount: 0.01,
           label: 'some promo',
-          source: promotion_action,
+          source: nil,
```

Comment:
Just one doubt: on line 172, it still mentions a 0.01 promotion adjustment. How is it possible if there's no promotion linked anymore here?

Reply:
Promotions without a source are "manual" adjustments, which work the same as promotion adjustments. They get applied before taxes.